File: src/org/baksia/rustycage/editors/RustPartitionScanner.java
package org.baksia.rustycage.editors;

import org.eclipse.jface.text.rules.*;

public class RustPartitionScanner extends RuleBasedPartitionScanner {
    public static final String RUST_COMMENT = "__rust_comment";

    public RustPartitionScanner() {
        IToken comment = new Token(RUST_COMMENT);
        IPredicateRule[] rules = new IPredicateRule[1];
        rules[0] = new MultiLineRule("/*", "*/", comment);
        setPredicateRules(rules);
    }
}

File: rustycage-plugin/org/baksia/rustycage/editors/RustPartitionScanner.scala
package org.rustycage.editors


class RustPartitionScanner extends RuleBasedPartitionScanner {
  val comment = new Token("__rust_comment")
  val rules: Array[IPredicateRule] = Array(new MultiLineRule("/*", "*/", comment))
  setPredicateRules(rules)
}